octodns/manager.py

```diff
@@ -111,30 +112,76 @@ def __init__(self, config_file, max_workers=None, include_meta=False):
             '__init__: config_file=%s (octoDNS %s)', config_file, version
         )
 
+        self._configured_sub_zones = None
+
         # Read our config file
         with open(config_file, 'r') as fh:
             self.config = safe_load(fh, enforce_order=False)
 
+        zones = self.config['zones']
+        self.config['zones'] = self._config_zones(zones)
```

Comment:
This bit is new, before zones was just the `dict` as loaded by `safe_loads`, see the method for more info.